src/lib/api/group/client/groupApiClient.client.ts

```diff
@@ -122,7 +122,7 @@ export const deleteMember = async (
 ): Promise<Result<undefined, string>> => {
   try {
     await request({
-      url: `${BACKEND_API_DOMAIN}/groups/${groupId}/users/${userId}`,
+      url: `${API_BASE_URL}/groups/${groupId}/users/${userId}`,
```

Comment:
Found out this`BACKEND_API_DOMAIN` variable (which doesn't exist) is set as base url in here, and caused build error. The functionality shouldn't be also working tho, don't know how it's in develop branch now...